app/src/docs/v1.api-spec.yaml

```diff
@@ -309,32 +480,55 @@ components:
             - low
             - high
           example: normal
+        subject:
+          type: string
+          description: The email subject
+          example: 'Hello {{ someone }}'
+    MergeResponse:
+      type: array
+      items:
+        $ref: '#/components/schemas/TransactionResponse'
+    MessageAssociation:
+      type: object
+      required:
+        - msgId
+        - to
+      properties:
+        msgId:
+          type: string
+          description: A corresponding message uuid
+          format: uuid
+          example: 00000000-0000-0000-0000-000000000000
         to:
```

Comment:
I wonder if we can make this without returning 'to'.  If we can ensure that our array of msgId comes back in the exact same order that the request was made, the caller can map together based on array position?

Reply:
It's probably possible if we don't use a Promise.all(...) to aggregate the results together, though I'm not sure if we should be masking even that away.